tests/test_check_package_resolved_policy.py

```
@@ -0,0 +1,285 @@
+def remove_remote_package(repo):
+    write_text(shell_ui_manifest(repo), manifest("CmuxMobileShellUI", []))
+
+
+def bump_remote_requirement(repo):
+    write_text(
+        shell_ui_manifest(repo),
+        manifest(
+            "CmuxMobileShellUI",
+            [
+                '.package(url: "https://github.com/apple/swift-collections.git", '
+                'from: "2.0.0")'
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)